### ISSUE TITLE
Unable to submit /wp-admin/admin.php?page=integrations

### ISSUE THREAD

**Issue by jlyon** [proudcity/wp-proudcity#2044]
It looks like clicking Save on /wp-admin/admin.php?page=integrations no longer submits the form, it just refreshes the Fontawesome field (a refresh icon temporarily appears next to that field)

**Comment by curtismchale** [comment]
@jlyon yeah I made a silly mistake and targeted `button` instead of a proper variable. Fixed on master now.
